lib/active_encode/output.rb

```diff
@@ -0,0 +1,9 @@
+module ActiveEncode
+  class Output
+    include Status
+    include TechnicalMetadata
+
+    attr_accessor :id
+    attr_accessor :url
+  end
+end
```

Comment:
I see that now we move the tech metadata into input/output, instead of having it in core. is the input metadata a subset of the output metadata?

Reply:
Not really.  The technical metadata on the input is about the input file whereas the technical metadata on outputs is about the derivative files.  There might be a lot of duplicate information between input and outputs but it is important to track it separately since there might be differences (e.g. small differences in duration).